### ISSUE TITLE
Postman should send notifications to a specific Slack Channel

### ISSUE THREAD

**Issue by vinitajag** [a85/POSTMan-Chrome-Extension#10613]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request

### Is your feature request related to a problem?

We have created a Public Workspace on Postman and want comments received on the Workspace to be relayed to a dedicated Slack channel on our official Slack workspace. 

We followed this article - https://learning.postman.com/docs/integrations/available-integrations/slack/#receive-postman-notifications-in-slack. However, it helped to only add Postman as an app on Slack.

![image](https://user-images.githubusercontent.com/49394949/151510153-ca48044a-a0c9-4cb7-b653-e8b13d3c20e6.png)

### Describe the solution you'd like

It would be very helpful to receive the notifications on a specific Slack channel. This would help us assign team members to triage comments, analyse them and respond to the original poster on a near real-time basis.

![image](https://user-images.githubusercontent.com/49394949/151510175-5a54cdf3-76ac-4e51-8aa0-74ff5bb15a86.png)

**Comment by abhijitkane** [comment]
Thanks @vinitajag. Does the team currently use the activity feed on the 'workspace overview' tab?

**Comment by vinitajag** [comment]
> Thanks @vinitajag. Does the team currently use the activity feed on the 'workspace overview' tab?

No, we do not use the activity feed. Also, since we are on a free plan at the moment, we have a 3-user limit.

**Comment by megan-schumann** [comment]
Hi @vinitajag are you still interested in this feature? As you may have seen in recent Postcon announcements work is underway on the Slack integration. Once those features are available it would be great to hear your feedback on how it's working for your organization's needs!

**Comment by megan-schumann** [comment]
Hi @vinitajag have you had a chance to try the feature where you can connect a Postman workspace to a Slack channel to get updates from that workspace? Curious whether that is meeting your team's needs. 

Documentation here: https://learning.postman.com/docs/integrations/available-integrations/slack/slack-app/#connect-a-postman-workspace-to-slack
